db/models/user.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var User = sequelize.define('User', {
+    email: {
```

Comment:
Do we need id in the user model?

Reply:
Sequelize will add this automatically when sync'ing the table (along with created/updated fields). We can be more explicit about this.